### ISSUE TITLE
Insert Picture ability

### ISSUE THREAD

**Issue by millingjon** [byhestia/springseed#108]
I take a lot of notes and I was wondering if adding pictures to notes would be awesome!

**Comment by thakurvijendar** [comment]
yeah. having a feature of inserting images would be of great help..
a picture can explain a thousand words,
